Makefile

```diff
@@ -210,39 +210,7 @@ stop_postgres:
 
 ##############################################  PYTHON TESTS ###########################################################
 pytest:
-	-rm -r .featureform
-	curl -C - https://featureform-demo-files.s3.amazonaws.com/transactions_short.csv -o transactions.csv
-	python -m pytest client/tests/local_dash_test.py
-	python -m pytest client/tests/status_test.py
-	python -m pytest client/tests/test_cli.py
-	python -m pytest client/tests/resources_test.py
-	python -m pytest client/tests/register_test.py
-	python -m pytest client/tests/provider_config_test.py
-	python -m pytest client/tests/serving_test.py
-	python -m pytest client/tests/redefined_test.py
-#	python -m pytest client/tests/local_test.py
-	python -m pytest client/tests/localmode_quickstart_test.py
-	python -m pytest client/tests/register_test.py
-	python -m pytest client/tests/test_spark_provider.py
-	python -m pytest client/tests/test_localmode_include_label_ts.py
-	python -m pytest client/tests/test_localmode_lag_features.py
-	python -m pytest client/tests/test_localmode_caching.py
-	python -m pytest client/tests/test_metadata_repository.py
-	python -m pytest client/tests/test_parse.py
-	python -m pytest client/tests/test_autogenerated_variants.py
-	python -m pytest -m 'local' client/tests/test_serving_model.py
-	python -m pytest -m 'local' client/tests/test_getting_model.py
-	python -m pytest -m 'local' client/tests/test_search.py
-	python -m pytest -m 'local' client/tests/test_tags_and_properties.py
-	python -m pytest -m 'local' client/tests/test_class_api.py
-	python -m pytest -m 'local' client/tests/test_ondemand_features.py
-	python -m pytest -m 'local' client/tests/test_resource_registration.py
-	python -m pytest -m 'local' client/tests/test_source_dataframe.py
-	python -m pytest -m 'local' client/tests/test_training_set_dataframe.py
-	python -m pytest -m 'local' client/tests/get_provider_test.py
-	python -m pytest client/tests/test_deploy.py
-	-rm -r .featureform
-	-rm -f transactions.csv
+	python -m pytest
```

Comment:
what happens with the local and server stuff?

Reply:
So by default now it runs the local with that addopts flag in the conf file. You can run hosted and docker manually by running it with the those flags but we dont do that here